### DISCUSSION TITLE
NPM organization

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-cdt-cloud/cdt-cloud#18] [Q&A] [answered]
Hi fellow cdt-clouders! 

I see we have a `@eclipse-cdt-cloud` org/namespace on npmjs - https://www.npmjs.com/org/eclipse-cdt-cloud - and that the `eclipse-cloud-bot` is publishing to there. But I don't see where the job is to do that? Is it a GitHub action, or a Jenkins job?

I want to publish cdt-gdb-adapter and related code to the org as that will help consumers of it.

Thanks for any input.

**Comment by planger** [comment, marked as answer]
Hi @jonahgraham,

This is done via Github actions. Here is the publishing step defined:
https://github.com/eclipse-cdt-cloud/clangd-contexts/blob/main/.github/workflows/cd.yml

The Eclipse Foundation controls the `eclipse-cloud-bot` and created a publishing token that they make available as a Github secrect under `secrets.GITHUB_TOKEN`. I think this secret needs to be added per repository though. So I think you'll need to file a request that this secret is being added to the `cdt-gbd-adapter` repo, but then you can copy over the workflow and it should work.
